src/CoreMetadata/CoreMetadataHeader.jsx

```diff
@@ -63,12 +45,16 @@ class CoreMetadataHeader extends Component {
   render() {
     if (this.props.metadata) {
       // display the download button if the user can download this file
-      const { user, projectAvail } = this.props;
+      const { projectAvail } = this.props;
       const projectId = this.props.metadata.project_id;
-      const canDownload = canUserDownload(user, projectAvail, projectId);
       let downloadButton = null;
       let signedURLButton = null;
-      if (canDownload) {
+
+      if (
+        !useArboristUI
+        || userHasMethodOnProject('read-storage', projectId, this.props.userAuthMapping)
+        || projectIsOpenData(projectAvail, projectId)
+      ) {
```

Comment:
can you add a bit of comment here explaining that this button will always render if `useArboristUI === false` so other people don't need to search for clues from all the PR comments?

Reply:
Yes, thanks!! Also removing the outdated comment a few lines prior, which I completely missed before this 🤦‍♀